bids/README.md

```diff
@@ -23,41 +23,62 @@ The neurospin_to_bids.py script can be use without install the unicog module.
 The test_dataset follows a minimal set of conventions for the download to work.
 
 
-# Using:
+# Basic usage:
 
-        cd path_to/test_dataset
+        cd <path_to/test_dataset>
         python path_to/neurospin_to_bids.py -neurospin_database trio
 
+In this case, the **\<path_to/test_dataset\>** directory contains a **exp_info** directory with 2 files 
+(**participants.tsv** and **download.tsv**) indicating what are the subjects to download.
+For more information, read the Additional information section.
+Indicate your **neurospin_database** either "prisma" or "trio" (prisma is the default parameter).
```

Comment:
Actually "The neurospin_database option" as is written in the advanced usage sounds better